### ISSUE TITLE
Draft specification for model of changes in Bd load following translocation

### ISSUE THREAD

**Issue by rolandknapp** [SNARL1/translocation#55]
This model is described in the section, Materials and Methods - Frog population recovery - Changes in Bd load following translocation. Alternatively, because we never describe results of this model in the Results section, we could drop this model.

**Comment by mbjoseph** [comment]
Good catch. Given that we don't describe the results, perhaps it's easier to drop the model description from the methods section. Sound like the right call to you, or do you think we should write up some of the results?

**Comment by rolandknapp** [comment]
I'm fine with dropping description of the Bd model from the methods section. The model results don't add much to the story, so I don't see any good justification for including the description or results. A qualitative description of the results should suffice.
